src/Chapter2.hs

```diff
@@ -620,7 +651,11 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven [] = []
+takeEven [x] = [x]
+takeEven [x, _] = [x]
```

Comment:
This case is actually redundant, as it's covered by the next line.
Note, that `[x, _]` is syntax sugar for `(x : _ : []`).
Though, it is very helpful to cover smaller cases before generalizing :slightly_smiling_face: